backup/setup/migrations.json

```diff
@@ -1026,5 +1026,16 @@
 		"$date": "2021-06-16T13:13:38.703Z"
 	},
 	"__v": 0
+},
+{
+	"_id": {
+		"$oid": "60d1c6d0b2616a2aedc4bfd7"
+	},
+	"state": "up",
+	"name": "start_new_school_year_brb",
```

Comment:
how to name it next year?

Reply:
I hope we build this as a feature, maybe in SHD by then